### PR TITLE
Add an RPC to get currently logged in fingerprint

### PR DESCRIPTION
**Pull request by Quexington**

### CODE REVIEW

**Review by mariano54 — COMMENTED** [review]
Can we add a test?
What happens if nothing is logged in?

**Comment by Quexington** [comment]
> Can we add a test? What happens if nothing is logged in?

I ran a test in a branch that actually has tests for the RPC but this branch doesn't have them yet.  If nothing is logged in, it will return `None` which should come out to `null` on the javascript side I believe.

File: chia/rpc/wallet_rpc_api.py [hunk 1]
```diff
@@ -52,6 +52,7 @@ def get_routes(self) -> Dict[str, Callable]:
         return {
             # Key management
             "/log_in": self.log_in,
+            "/get_fingerprint": self.get_fingerprint,
```
Comment (paninaro) [inline chia/rpc/wallet_rpc_api.py:55]: Do we feel that the name `get_fingerprint` provides enough context to indicate that it returns the logged-in key's fingerprint? The name strikes me as being a bit general. If I were looking at the available APIs at a glance, I don't think my first thought would be that this indicates the currently logged-in wallet key. I could be an outlier though.

Comment (Quexington) [reply chia/rpc/wallet_rpc_api.py]: I changed the name to `get_logged_in_fingerprint` :+1:

**Review by wjblanke — APPROVED** [review]
aok

**Pull request merged**